otter/convergence/service.py

```diff
@@ -607,7 +607,7 @@ def buckets_acquired(self, my_buckets):
         """
         ceff = Effect(GetChildren(CONVERGENCE_DIRTY_DIR)).on(
             partial(self._converge_all, my_buckets))
-        return perform(self._dispatcher, self._with_conv_runid(ceff))
+        return (perform(self._dispatcher, self._with_conv_runid(ceff)), )
```

Comment:
Out of curiosity, does this even need to return at all?  Is the result used for anything except testing?